ci/create_yum_repo/yumrepo.j2

```diff
@@ -0,0 +1,5 @@
+[automotive]
+name = CentOS 8 Stream Automotive (RPMs)
```

Comment:
Maybe we could put here a sorted name so it's easier to use at `dnf/yum`.

The name at the `[]` is usually the long name and at the field `name` the sort one. (I think that) usually without spaces.

Reply:
I have looked at some of the repos on my system. Here is one `[hashicorp-test]
name=Hashicorp Test - $basearch
baseurl=https://rpm.releases.hashicorp.com/RHEL/7/$basearch/test
enabled=0
gpgcheck=1
gpgkey=https://rpm.releases.hashicorp.com/gpg`

Reply:
That's true, it's exactly the opposite of what I told you :sweat_smile: